schema.graphql

```diff
@@ -12,6 +12,8 @@ type Project {
   publicId: UUID!
   parentId: UUID
   title: String!
+  description: String!
```

Comment:
Will this solution work for existing projects, where this field is empty? :thinking:

Reply:
I don't believe backwards compatibility or migrating old data was mentioned as a requirement for this flip.

Reply:
Correct me if I am wrong, but you are saying that if we merge this PR and then deploy it all projects created prior this moment will stop working? 🤔

Reply:
@MaxStalker sorry if there is any misunderstanding here. I believe simply making the title, description and readme as not required fields, so we remove the exclamation point, then old projects will work. Should we check with Pete about this?

Reply:
@justjoolz is right - FLIP description doesn't have the requirement for migration/compatibility.
We can handle this before merging this PR into master.

Personally, I was just wondering if you considered how your code would affect old deployed projects.

Reply:
Marking this complete @MaxStalker -- 'title', 'description' and 'readme' are now optional data members in the schema with my last commit and based on our chat about this. Marking this as resolved, tho happy to circle back after Pete provides guidance.